Library/OcBootManagementLib/BootEntryManagement.c

```diff
@@ -2548,6 +2548,10 @@ OcLoadBootEntry (
       // Unload dmg if any.
       //
       InternalUnloadDmg (&DmgLoadContext);
+      //
+      // Unload image.
+      //
+      gBS->UnloadImage (EntryHandle);
```

Comment:
Not sure if this gBS->UnloadImage call needs further wrapping, e.g. make a new Context->UnloadImage, or if that's only worth adding if there's anything special for it to do, above and beyond just the gBS call.